execution/interface.go

```diff
@@ -68,7 +68,7 @@ type FullExecutionClient interface {
 	Start(ctx context.Context) error
 	StopAndWait()
 
-	Maintenance() error
+	Maintenance(capLimit int64) error
```

Comment:
this should be a config parameter inside the execution client and not a parameter passed from consensus

Reply:
Done.